db/commands/Train.js

```diff
@@ -17,26 +17,48 @@ const Train = {
     return  knex.select('*')
     .where({ id: id })
     .from('train')
+    .catch( () => {
+      console.log('ERROR: Train does not exists in database.')
+      return 'Train does not exist.'
+    })
     .then( train => train[0] )
   },
 
   getCurrentLocation: ( id ) => {
-    return knex.column('location').select().where({ id: id }).from('train').then( location => location[0].location )
+    return knex.column('location')
+    .select()
+    .where({
+      id: id
+    })
+    .from('train')
+    .then( location => location[0].location )
   },
 
   getNextLocation: ( id ) => {
     return Train.getCurrentLocation( id )
     .then( currentLocation => {
-     let currentIndex = stations.indexOf( currentLocation )
-     const nextIndex = currentIndex === 11 ? 0 : currentIndex + 1
-     const nextLocation = stations[ nextIndex ]
-     return nextLocation
+      let currentIndex = stations.indexOf( currentLocation )
+      const nextIndex = currentIndex === 11 ? 0 : currentIndex + 1
+      const nextLocation = stations[ nextIndex ]
+      return nextLocation
    })
   },
 
   gotoNextLocation: ( id ) => {
```

Comment:
The function name is misleading to me. Looks like you are updating the location of the train, so you should name it to something that shows the intent of the update.